src/test/java/com/google/cloud/pubsublite/kafka/PubsubLiteConsumerTest.java

```diff
@@ -481,6 +478,22 @@ public void endOffsets() {
     assertThat(output).isEqualTo(ImmutableMap.of(partition2, 22L, partition4, 44L));
   }
 
+  @Test
+  public void offsetsForTimes() {
+    TopicPartition partition2 = new TopicPartition(example(TopicPath.class).toString(), 2);
+    TopicPartition partition4 = new TopicPartition(example(TopicPath.class).toString(), 4);
+    when(topicStatsClient.computeCursorForEventTime(
+            example(TopicPath.class), Partition.of(2), Timestamps.fromMillis(2000)))
+        .thenReturn(
```

Comment:
please test the nullability behavior of this

Reply:
`partition4` below tests the null case.